modules/permutiveRtdProvider.js

```diff
@@ -84,23 +85,34 @@ export function setBidderRtb (auctionDetails, customModuleConfig) {
 }
 
 /**
- * Merges segments into existing bidder config
+ * Merges segment data into existing bidder config
+ * Segments are retrieved from the `ac` property of `segmentData`
  * @param {Object} currConfig - Current bidder config
- * @param {Object} segmentData - Segment data
+ * @param {Object[]} transformationConfigs - array of objects with `id` and `config` properties, used to determine
+ *                                           the transformations on user data to include the ORTB2 object
+ * @param {string[]} segmentIDs - Permutive segment IDs
  * @return {Object} Merged ortb2 object
  */
-function mergeOrtbConfig (currConfig, segmentData) {
-  const segment = segmentData.ac.map(seg => {
-    return { id: seg }
-  })
+function mergeOrtbConfig (currConfig, segmentIDs, transformationConfigs) {
```

Comment:
There's a utils finding for this now, i think it's called mergeDeep

Reply:
Thanks for the suggestion – after looking at this function I realised it's not really doing a simple merge of two objects but actually updating the `user.data` array with some new entries. So I don't think `mergeDeep` is suitable here, but I've renamed the function to clarify what it does.